roles/pulp/defaults/main.yml

```diff
@@ -24,7 +24,7 @@ pulp_upgrade: false
 #  they are confident that this version of pulp_installer is compatible with
 #  pulp_version.
 #  Ignored if pulp_source_dir is set
-pulp_version: "3.3.1"
+pulp_version: "3.5.0.dev"
```

Comment:
Like we discussed at the meeting, this string is for PyPI installs. So it needs to be set to the current stable release.

Reply:
As part of the 3.4.0 release, you should set it to 3.4.0.